packages/server/graphql/mutations/helpers/notifications/NotificationHelper.ts

```diff
@@ -0,0 +1,96 @@
+import {IntegrationProviderMattermost} from '../../../../postgres/queries/getIntegrationProvidersByIds'
+import Meeting from '../../../../database/types/Meeting'
+import {SlackNotificationEvent} from '../../../../database/types/SlackNotification'
+import {DataLoaderWorker} from '../../../graphql'
+import {MattermostNotificationHelper} from './MattermostNotificationHelper'
+import {SlackNotificationHelper} from './SlackNotificationHelper'
+
+async function getSlack(
+  dataLoader: DataLoaderWorker,
+  event: SlackNotificationEvent,
+  teamId: string
+) {
+  const notifications = await dataLoader
+    .get('slackNotificationsByTeamIdAndEvent')
+    .load({event, teamId})
+  return notifications.map(SlackNotificationHelper)
+}
+
+async function getMattermost(dataLoader: DataLoaderWorker, meeting: Meeting, teamId: string) {
+  const {facilitatorUserId} = meeting
+  const provider = await dataLoader
+    .get('bestTeamIntegrationProviders')
+    .load({service: 'mattermost', teamId, userId: facilitatorUserId})
+  return provider ? [MattermostNotificationHelper(provider as IntegrationProviderMattermost)] : []
+}
+
+async function loadMeetingTeam(dataLoader: DataLoaderWorker, meetingId: string, teamId: string) {
+  const [team, meeting] = await Promise.all([
+    dataLoader.get('teams').load(teamId),
+    dataLoader.get('newMeetings').load(meetingId)
+  ])
+  return {
+    meeting,
+    team
+  }
+}
+
+async function getAllNotifiers(
+  dataLoader: DataLoaderWorker,
+  event: SlackNotificationEvent,
+  meeting: Meeting,
+  teamId: string
+) {
+  const [slack, mattermost] = await Promise.all([
+    getSlack(dataLoader, event, teamId),
+    getMattermost(dataLoader, meeting, teamId)
+  ])
+
+  return [...slack, ...mattermost]
+}
+
+export const NotificationHelper = {
+  // returning void because otherwise errors would need to be picked apart per integration and this manager should deal with retries
+  async startMeeting(dataLoader: DataLoaderWorker, meetingId: string, teamId: string) {
```

Comment:
+2 I love how succinct all of the functions are in this file. It's so easy to read